packages/core/admin/package.json

```diff
@@ -101,6 +101,7 @@
     "mini-css-extract-plugin": "2.4.4",
     "moment": "^2.29.1",
     "node-polyfill-webpack-plugin": "1.1.4",
+    "object-assign": "4.1.1",
```

Comment:
This one is a polyfill as well so we might not need this one. Do you know which lib needs it?
Could you remove it from the webpack.alias.js file?

Reply:
I don't now where it comes from. I made the changes although I then bumped into many other errors when building the UI, so it's likely yarn 2 support is not possible for now

Reply:
@cortopy we're using Yarn 2 with PnP and Strapi. We do get all of the module warnings, but we're able to build and run successfully. What errors were you running into?

If this PR is dead in the water, my team may take a stab at updating it. Yarn PnP is amazing, and explicitly naming missing dependencies would be great for Strapi